.storybook/main.js

```diff
@@ -1,5 +1,5 @@
-const customWebpack = require('../webpack.dev.config.js');
-const customWebpackCSS = require('../webpack.css.dev.config.js');
+const customWebpack = require('../webpack.config.js');
```

Comment:
I don't think that we are currently using the imported webpack in this configuration.  It might be worth verifying that and removing it if we are not using it.  There is a way to build upon a custom webpack configuration.  Feel free to experiment with that to see if there is any value in doing it that way.  

https://storybook.js.org/docs/react/configure/webpack

Otherwise, remove the imports to avoid confusion if they have no impact.